xml/schema/CRM/CiviGeometry/GeometryEntity.xml

```diff
@@ -50,5 +50,15 @@
     <fieldName>entity_table</fieldName>
     <unique>true</unique>
   </index>
-
+  <field>
+    <name>expiry_date</name>
+    <type>timestamp</type>
```

Comment:
@johntwyman @unfinishedteleporter I specifically used timestamp here because that will mean that if someone sets an expiry in WA to 10am on 3 Jul 2021 say, Then the cron job that might be running on east coast won't remove it until 12pm East Coast time as timestamp takes into account timezone matters